web_poet/pages.py

```diff
@@ -100,15 +100,20 @@ def __init_subclass__(cls, skip_nonitem_fields=_NOT_SET, **kwargs):
             return
         cls._skip_nonitem_fields = skip_nonitem_fields
 
-    @validates_input
     async def to_item(self) -> ItemT:
-        """Extract an item from a web page"""
+        """Extract an item"""
         return await item_from_fields(
             self,
             item_cls=self.item_cls,
             skip_nonitem_fields=self._get_skip_nonitem_fields(),
         )
 
+
+class ItemPage(Extractor[ItemT], Injectable):
+    """Base Page Object, with a default :meth:`to_item` implementation
+    which supports web-poet fields.
+    """
```

Comment:
I would refrain mentioning the `to_item` implementation, since it basically comes from Extractor.

```suggestion
    """Base class for page objects."""
```